TicketController

```diff
@@ -0,0 +1,127 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+
+//Ticket controller is used to add tickets to tickets database
+//Also used to update default ticket price value in prices database
+
+
+public class TicketController {
+	private String fileName;
+	private String priceFileName;
+	
+	
+	public TicketController(String ticket_file_name,String prices_file_name){
+		fileName = ticket_file_name; //Input the directory where your .dat file is located
+		priceFileName = prices_file_name;
+	}
+	
+	public void updatePriceToDB(int moviename_index, int cinema_index,int showtime_index, int price) {
+		//do i have to use arraylist? 
```

Comment:
No need, array suffices for `writeObject` since it takes as argument an `Object` which can be literally anything, even an array. However, the format of writing may differ. If you write an entire array, all the values will be included on a single line, whereas if you sue a for loop and write to each line, that will write the array vertically. Depends on what read method you're using.